doc/source/devel/taurus3to4.rst

```diff
@@ -151,11 +151,11 @@ This list is a best-effort to document changes, but it may not be 100% complete
 +-----------------------------------------------+---------------------------------------------------------------+
 | TangoDevice.getSWState                        | TangoDevice.state                                             |
 +-----------------------------------------------+---------------------------------------------------------------+
-| TangoDevice.getValueObj                       | TangoDevice.stateObj.read  *tango* or                         |
-|                                               | state  *agnostic*                                             |
+| TangoDevice.getValueObj                       | TangoDevice.stateObj.read()  *tango* or                       |
```

Comment:
Hi @reszelaz, thanks for reviewing, but I do not think that this precise correction is right.
The replacement for the **method object** `TangoDevice.getValueObj` is the **method object** `TangoDevice.stateObj.read` (no parentheses)

This is the cryterium that we used when writing the table (although I am aware that we may not have been 100% consistent along the whole table)

Reply:
Sorry, I was mislead that in the context of this change there were a lot of `read()` calls. Looking at the whole table I see the general cryterium. I just reverted this change.